runtime/interpreter/value.go

```diff
@@ -6525,7 +6525,7 @@ type AccountValue interface {
 // AuthAccountValue
 type AuthAccountValue struct {
 	Address                 AddressValue
-	storageUsedGet          func() UInt64Value
+	storageUsedGet          func(interpreter *Interpreter) UInt64Value
 	storageCapacityGet      func() UInt64Value
```

Comment:
storageCapacityGet will possibly need the same thing, but we can wait with changing that until I confirm that is the case.